### ISSUE TITLE
get_bi_flows error

### ISSUE THREAD

**Issue by DachunKai** [huicongzhang/BSSTNet#3]
Hi, could you test the [get_bi_flows snippet](https://github.com/huicongzhang/BSSTNet/blob/main/basicsr/models/Video_BSST_model.py#L65-L78) in Video_BSST_model.py.

I tested it with following:
```python
import torch
lqs = torch.rand(1, 10, 3, 180, 320).cuda()
flows_forward, flows_backward = get_bi_flows(lqs)
```
But it turns error in [Line 66 ](https://github.com/huicongzhang/BSSTNet/blob/main/basicsr/archs/RAFT/utils/utils.py#L66) that 
`Exception has occurred: RuntimeError
grid_sampler(): expected grid and input to have same batch size, but got input with sizes [2160, 1, 12, 20] and grid with sizes [1980, 9, 9, 2]`

Thanks!

**Comment by DachunKai** [comment]
Moreover, could you provide a test case, especially the input tensor shape? What is the minimum resolution requirement for the input lqs? How can I properly run your [forward function in line 1029](https://github.com/huicongzhang/BSSTNet/blob/main/basicsr/archs/BSST_arch.py#L1029)? I encountered many issues with shape mismatches.

**Comment by Wenju-Huang** [comment]
My lq input shape is [1,24,3,1920,1080], and also meet this problem.

**Comment by DachunKai** [comment]
@huicongzhang Hi，how to solve this problem? Thanks!

**Comment by huicongzhang** [comment]
First, the input shape is (b,t,3,h,w), and h and w need to be divisible by 8 in [get_bi_flows](https://github.com/huicongzhang/BSSTNet/blob/956fabb85801e2deac089a744b843571f22e633a/basicsr/models/Video_BSST_model.py#L65). Additionally, in the code implementation [test_by_patch](https://github.com/huicongzhang/BSSTNet/blob/956fabb85801e2deac089a744b843571f22e633a/basicsr/models/Video_BSST_model.py#L220) , I cropped the input to (b,t,3,256,256) and input it into the [forward function](https://github.com/huicongzhang/BSSTNet/blob/956fabb85801e2deac089a744b843571f22e633a/basicsr/archs/BSST_arch.py#L1029) for inference. Therefore, the input shape for the [forward function](https://github.com/huicongzhang/BSSTNet/blob/956fabb85801e2deac089a744b843571f22e633a/basicsr/archs/BSST_arch.py#L1029) should be (b,t,256,256).

**Comment by DachunKai** [comment]
@huicongzhang Thanks! If I want to inference a video with shape (10, 3, 180,320)，and the output shape should also be (10, 3, 180,320). Can the current model achieve it?

**Comment by huicongzhang** [comment]
@DachunKai You can pad the input LQs to the shape (10, 3, 256, 320) using the padding function before inputting them into     [test_by_patch](https://github.com/huicongzhang/BSSTNet/blob/956fabb85801e2deac089a744b843571f22e633a/basicsr/models/Video_BSST_model.py#L220).

**Comment by DachunKai** [comment]
@huicongzhang Hi, Huicong. I found that the input LQs shape (b, t, 3, 256, 256) has limits on the dimension 't', where 't' should be a number greater than or equal to 4. It doesn't work when t=1, 2, or 3. Do you know why this might be?

**Comment by YngMgC** [comment]
> 我的lq输入形状是[1,24,3,1920,1080]，也遇到这个问题。

Hi! I met the same question, have you solve it yet?
